Add bind-tools option to ease the server/tools package split

--- meson.options.orig	2025-11-19 15:19:33 UTC
+++ meson.options
@@ -203,3 +203,5 @@ option(
     value: 'thin',
     description: 'Enable Link Time Optimization for named.',
 )
+
+option('bind-tools', type: 'feature', value: 'disabled', description: 'build tools only')
